source/includes/stackpath/_workloads.md

```diff
@@ -117,6 +117,103 @@ Attributes | &nbsp;
 `isRemoteManagementEnabled`<br/>*boolean* | Specify if remote management is enabled on workload instance or not.
 `image`<br/>*string* | The workload's instance operating system image.
 
+<!-------------------- CREATE A WORKLOAD -------------------->
+### Create a workload
+```shell
+curl -X POST \
+    -H "MC-Api-Key: your_api_key" \
+    -d "request_body" \
+    "https://cloudmc_endpoint/v1/services/stackpath/test-area/workloads"
+```
+> Request body example for a VM:
```

Comment:
Just a small formatting change to match [our docs template](https://raw.githubusercontent.com/cloudops/cloudmc-api-docs/cmc-dev/.github/PULL_REQUEST_TEMPLATE.md) 📜 

To be consistent with the rest of the docs, we could put both request examples in the same code block (with format js) and the response example in another.

> Request body examples:
```js
// create a vm
{
   name: my-vm,
   slug: my-vm-slug,
   ...
}

// create a container
{
   name: my-container,
   slug: my-container-slug,
   ...
}
```

> The above commands return a JSON structured like this:
```json
{
  "taskId": "ef70cafa-0544-4709-a66a-c68595ee105a",
  "taskStatus": "SUCCESS"
}

```

Reply:
fixed in next commit